deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -78,9 +75,6 @@ metadata:
   name: cephfs-external-provisioner-cfg
 rules:
   # remove this once we stop supporting v1.0.0
```

Comment:
does this comment not refer to the ConfigMaps section? if that is the case, remove it as well

Reply:
yep, removed that as well.